test/runtests.jl

```diff
@@ -99,28 +105,25 @@ end
 
 function validparents(node::IntervalTrees.InternalNode)
     for child in node.children
-        if !validparents(child) || get(child.parent) != node
+        if !validparents(child) || child.parent != node
             return false
         end
     end
     return true
 end
 
-function validparents(node::IntervalTrees.LeafNode)
-    return true
-end
-
+validparents(node::IntervalTrees.LeafNode) = true
 
 # Verify that sibling/cousin pointers are correct
 function validsiblings(t::IntervalTrees.IntervalBTree)
-    if !isnull(t.root.left) || !isnull(t.root.right)
+    if !(t.root.left === nothing) || !(t.root.right === nothing)
```

Comment:
Could be a lot simpler as:
`t.root.left === t.root.right === nothing || return false`

Reply:
agreed will do the change. One of the reasons I did not change the code significantly was because I felt the person who wrote it may not be able to review it and others may object to significant logical changes.

Reply:
https://github.com/BioJulia/IntervalTrees.jl/pull/37/commits/bfe59b002e5f687d99faf5a65c1435efface8379 has the changes as suggested.